### PR TITLE
SWTCH-1270 create a role with other role as an issuer

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

**Review by jrhender — CHANGES_REQUESTED** [review]
Getting an error: `Cannot read properties of undefined (reading 'issuer')`
image

**Comment by dawidgil** [comment]
Updated. Should be good now.

**Comment by jrhender** [comment]
@dawidgil better, but still not quite working for me. I'm not sure if this was there before but after I set a role to have a role as an issuer, I can't go back and edit the role https://www.loom.com/share/d7cd24c3eef7405e826484957483fd15

**Comment by dawidgil** [comment]
ok... i will look on it. i checked it and it was working before :/

**Comment by dawidgil** [comment]
ah, i watched this now. This is bug that i mentioned to you yesterday. I'm aware of it. This bug wasn't previously because this feature wasn't working 😈 . Right now just this information is not updated.
I mentioned that this one feature need refactoring + unit testing to be sure that it's working. However i will try to fix it on Monday.

**Review by Malak67 — APPROVED** [review]

File: src/app/routes/applications/governance-view/governance-details/governance-details.component.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
           <span class="color-white mb-0"> {{ formData?.definition?.issuer?.issuerType }} </span>
         </div>
         <mat-divider></mat-divider>
-        <ng-container *ngIf="formData?.definition?.issuer?.issuerType === 'Role'">
+        <ng-container *ngIf="formData?.definition?.issuer?.issuerType === 'ROLE'">
```
[the remark this answers is not in the document]
Comment (dawidgil) [reply src/app/routes/applications/governance-view/governance-details/governance-details.component.html]: adjusted + I added few more methods and pipe :)

**Review by jrhender — APPROVED** [review]
Test locally. Looks great!

**Pull request merged**